Makefile

```diff
@@ -221,46 +192,34 @@ endif
 	gzip -9 < $^ > $@.tmp
 	mv $@.tmp $@
 
-nixpkgs/packages-nixos-$(NIXOS_SERIES).json: packages-config.nix
-	nixpkgs=$$(nix-instantiate --find-file nixpkgs -I nixpkgs=$(CHANNEL_NIXOS_STABLE)); \
-	(echo -n '{ "commit": "' && (cat $$nixpkgs/.git-revision || printf "unknown") && echo -n '","packages":' \
-	  && nix-env -f '<nixpkgs>' -I nixpkgs=$(CHANNEL_NIXOS_STABLE) -qa --json --arg config 'import ./packages-config.nix' \
-	  && echo -n '}') \
-	  | sed "s|$$nixpkgs/||g" | jq -c . > $@.tmp
-	python -mjson.tool < $@.tmp > /dev/null
-	mv $@.tmp $@
+.PHONY: nixpkgs/packages-nixos-$(NIXOS_SERIES).json nixpkgs/packages-nixos-$(NIXOS_SERIES).json.gz
 
-nixpkgs/packages-nixos-unstable.json: packages-config.nix
-	nixpkgs=$$(nix-instantiate --find-file nixpkgs -I nixpkgs=$(CHANNEL_NIXOS_UNSTABLE)); \
-	(echo -n '{ "commit": "' && (cat $$nixpkgs/.git-revision || printf "unknown") && echo -n '","packages":' \
-	  && nix-env -f '<nixpkgs>' -I nixpkgs=$(CHANNEL_NIXOS_UNSTABLE) -qa --json --arg config 'import ./packages-config.nix' \
-	  && echo -n '}') \
-	  | sed "s|$$nixpkgs/||g" | jq -c . > $@.tmp
-	python -mjson.tool < $@.tmp > /dev/null
-	mv $@.tmp $@
+nixpkgs/packages-nixos-$(NIXOS_SERIES).json:
+	@ln -sfn $(NIXPKGS_STABLE)/packages.json $@
 
-nixpkgs/packages-nixpkgs-unstable.json: packages-config.nix
-	nixpkgs=$$(nix-instantiate --find-file nixpkgs -I nixpkgs=$(CHANNEL_NIXPKGS_UNSTABLE)); \
-	(echo -n '{ "commit": "' && (cat $$nixpkgs/.git-revision || printf "unknown") && echo -n '","packages":' \
-	  && nix-env -f '<nixpkgs>' -I nixpkgs=$(CHANNEL_NIXPKGS_UNSTABLE) -qa --json --arg config 'import ./packages-config.nix' \
-	  && echo -n '}') \
-	  | sed "s|$$nixpkgs/||g" | jq -c . > $@.tmp
-	python -mjson.tool < $@.tmp > /dev/null
-	mv $@.tmp $@
+nixpkgs/packages-nixos-$(NIXOS_SERIES).json.gz:
+	@ln -sfn $(NIXPKGS_STABLE)/packages.json.gz $@
+
+.PHONY: nixpkgs/packages-nixpkgs-unstable.json nixpkgs/packages-nixpkgs-unstable.json.gz
+
+nixpkgs/packages-nixpkgs-unstable.json:
+	@ln -sfn $(NIXPKGS_UNSTABLE)/packages.json $@
+
+nixpkgs/packages-nixpkgs-unstable.json.gz:
+	@ln -sfn $(NIXPKGS_UNSTABLE)/packages.json.gz $@
+
+NIXOS_OPTIONS = /no-such-path
+
+.PHONY: nixos/options.json
 
 # Cute hack, this allows future expansion if desired
 # Mainly, this allows tracking NIXOS_SERIES
 nixpkgs/packages-channels.json: Makefile
-	echo '["nixos-$(NIXOS_SERIES)", "nixos-unstable", "nixpkgs-unstable"]' > $@
+	echo '["nixos-$(NIXOS_SERIES)", "nixpkgs-unstable"]' > $@
```

Comment:
Lately, we have had consistently fresher `nixos-unstable`, compared to `nixpkgs-unstable`. Should we prefer the channel that updates more regularly?